.vscode/launch.json

```diff
@@ -16,16 +16,16 @@
       "type": "extensionHost",
       "request": "launch",
       "runtimeExecutable": "${execPath}/client/client.js",
-      "args": ["--extensionDevelopmentPath=${workspaceFolder}"]
+      "args": ["--extensionDevelopmentPath=${workspaceFolder}/packages/language-server"]
```

Comment:
This changes do not needed as extension debugs from `hegel/packages/language-server` directory, so with this changes there will be double path.

Reply:
i have moved `.vscode/launch.json` in root directory of repo so `workspaceFolder` is not `hegel/packages/language-server` (it's root directory of repo)

Reply:
Is this goal of creating monorepo?

Reply:
well yes. if it's separate workspaces in IDE then what's a point in having single repository?

that is reason my i moved flowconfig in the root folder https://github.com/JSMonk/hegel/commit/0e760189c52b813e14c4c62622416e94fc4acbf0#diff-a9d0201e8c5bdc946e54218bb9206174

and i plan to have single eslint config, single hegelrc, etc.

that is what all other monorepos do as i know